assets/scss/bulmaswatch/_overrides.scss

```diff
@@ -0,0 +1,145 @@
+// Overrides
+@if $bulmaswatch-import-font {
+    @import url( 'https://fonts.googleapis.com/css?family=Muli:400,700');
```

Comment:
Expected 0 spaces after `(` instead of ` `

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#spacebetweenparens)